ui/src/utils/mittBus.ts

```diff
@@ -15,6 +18,12 @@ type Event = {
   'connect-terminal': customTreeOption;
   'set-theme': { themeName: string };
   'file-manage': { path: string; type: ManageTypes; new_name?: string };
+  'file-upload': {
+    fileList: Ref<Array<UploadFileInfo>>;
+    onFinish: () => void;
+    onError: () => void;
+    onProgress: (e: { percent: number }) => void;
+  };
   'download-file': { path: string; is_dir: boolean };
   'terminal-search': { keyword: string; type?: string };
   'share-user': { type: string; query: string };
```

Comment:
I see no immediate issues with the provided code. However, it's worth noting that the use of type definitions is crucial for clarity and type checking purposes if used in production, but this doesn't affect the functionality currently as there aren't any errors nor obvious inconsistencies.

Additionally, considering performance improvements, you might want to optimize:

* Check console logs regularly instead of only when an error occurs to catch potential issues early.
* Avoid creating unnecessary objects or variables unless they serve a specific purpose.
* Use const for properties where modification is not expected.
  
If any specific questions arise related to these points during further development processes, feel free to ask!